### ISSUE TITLE
UX: Risk View Countdown unclear

### ISSUE THREAD

**Issue by kosmar** [corona-warn-app/cwa-app-ios#719]
<img width="180" alt="Bildschirmfoto 2020-06-19 um 08 42 32" src="https://user-images.githubusercontent.com/1532381/85104518-0b2a6900-b209-11ea-8474-e03b67733f7f.png">

- it is unclear to user what the countdown display does convey and what might happen on day 14?
- detail text after tap does not answer this question
- i would love to propose an improvement, but i also don’t know the answer.

please, can you explain?

**Comment by max-hzi** [comment]
I agree this is something I was wondering about, too. From an epidemiological point of view, it makes sense to have the number of days since last exposure. 
For days without exposure, it is a little different. If the app is not active on any day within the past 14 days, exposure cannot be ruled out (in the context of the app). If the app is active for 14 days and did not detect any exposure, the user is considered safe (also looking from the app's perspective). I think the wording may be optimised here, maybe to 'days active within the past 14 days'. Status quo suggests that these 14 days are finite.
Can anyone confirm/ clarify?

**Comment by kosmar** [comment]
so the meaning is 
_full days of active app within last 14 days_
?
and if i turn bluetooth off for an hour i break the streak?

please clarify

**Comment by paulinepauli** [comment]
No, I turn bluetooth off each night and the count continues.

**Comment by kosmar** [comment]
_Tägliche Aktualisierung_ seems misleading or unclear, too
https://twitter.com/macava/status/1273884530252881920

**Comment by kosmar** [comment]
maybe to understand the issue better,  can you tell us what would currently be the countdown text displayed on day 15?

**Comment by kosmar** [comment]
additionally, its not transparent which _risk_ is implied here. 

is it telling me that i behave risky because the app learned that i meet more people than current stats in germany suggest? (would make most sense imho)

or is it the risk of infecting someone else, because i did encounter x positive people recently? (also makes sense, but very delayed and fuzzy)

**Comment by kosmar** [comment]
also
the view tells me on day 3 risk is 'low', but rather it should be 'unknown', as the app knows virtually nothing yet.

**Comment by paulinepauli** [comment]
Maybe the point is:
If the app has not run 14 out of 14 days, a complete risk level cannot be named.

But why do I have a low risk level with only 3 days at the moment?

**Comment by max-hzi** [comment]
> additionally, its not transparent which risk is implied here.

I suggest that you have a look at the documentation (e. g. over here https://github.com/corona-warn-app/cwa-documentation/blob/master/cwa-risk-assessment.md). It reports if **you** had contact with a person which reported a positive test result.
Your risk could indeed already be high, but only if the app recorded contact with a person which reported a positive test.

As a side notice, please try to reduce the number of postings and gather the comments in a single one.

**Comment by tkowark** [comment]
As this is also a cross platform issue, let's continue the discussion in the cwa-documentation repository.
